src/PopoverMenuItem.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router';
+import cx from 'classnames';
+
+/**
+ * @module PopoverMenuItem
+ */
+class PopoverMenuItem extends React.Component {
+
+	render() {
+		const {
+			children,
+			url,
```

Comment:
might be better to call this `to`, since it shouldn't be a fully-qualified URL, but rather a route/location. `to` also makes a better connection to the underlying `<Link>` interface